userguide/intro.rst

```diff
@@ -303,6 +303,13 @@ These screen options have changed:
 * :ref:`Directory Services Monitor <Directory Services>` has been added
   to the top toolbar row.
 
+* Disk type, rotation rate, and manufacturer information can be viewed
+  on the :ref:`Disks` page.
+
+* The :guilabel:`Disable LDAP user/group cache` checkbox has been added
```

Comment:
This should be title case, probably in the UI also.
```suggestion
* The :guilabel:`Disable LDAP User/Group Cache` checkbox has been added
```

Reply:
@wblock webUI PR: https://github.com/freenas/webui/pull/2648